CMakeLists.txt

```diff
@@ -20,29 +23,81 @@ add_compile_definitions(TFLITE_BUILD_WITH_XNNPACK_DELEGATE)
 
 # This assumes that tensorflow got checked out as submodule.
 # True, if `git clone` had the additional option `--recursive`.
-add_subdirectory(tensorflow/tensorflow/lite 
+add_subdirectory(tensorflow/tensorflow/lite
```

Comment:
Could we make this path dynamic, so that people can override this with a parameter to `cmake`?
This also allows to bail out if TFlite is not properly checked out/cannot be found.

Reply:
Sure - I'll use an overridable variable - might want this back in main too?